packages/app/src/app/pages/Dashboard/Components/VariableGrid/index.tsx

```diff
@@ -322,7 +322,12 @@ export const VariableGrid: React.FC<VariableGridProps> = ({
   });
 
   return (
-    <Element style={{ width: '100%', height: '100%' }}>
+    <Element
+      style={{
+        width: '100%',
+        height: '100%',
+      }}
+    >
       <AutoSizer onResize={onResize}>
         {({ width, height }) => {
           const cappedWith = Math.min(width, GRID_MAX_WIDTH);
```

Comment:
Should we take the adjusted width (`width + GUTTER * 2`) into account for the `cappedWidth`? Since it's what defines the number of columns?